libs/utils/utils.js

```diff
@@ -1042,6 +1042,7 @@ async function processSection(section, config, isDoc) {
     const { default: loadInlineFrags } = await import('../blocks/fragment/fragment.js');
     const fragPromises = inlineFrags.map((link) => loadInlineFrags(link));
     await Promise.all(fragPromises);
+    await decoratePlaceholders(section.el, config);
```

Comment:
Wouldn't we want to replace placeholder in fragments as a rule? Can't this be added as part of `fragments.js` so that any loaded fragment comes with placeholders already replaced?

Reply:
That is a very good catch actually - placeholders should also be replaced when the function is consumed standalone

Reply:
I thought that was already the case. Didn't really check, I agree  all fragments should be processed with placeholders. I'll take a look.

Reply:
@overmyheadandbody I think the title is misleading and this fix will also apply to fragments.

Reply:
It's not clear to me how placeholders are replaced in https://github.com/adobecom/milo/blob/stage/libs/blocks/fragment/fragment.js. Other pieces of logic might use `fragment.js` directly, so the logic here in `utils.js` will not apply. In my view, the fragment logic should handle placeholder replacement OOTB

Reply:
@overmyheadandbody normal fragments are processed with `loadArea` and placeholders are already processed therein.